.storybook-s2/docs/Migrating.jsx

```diff
@@ -0,0 +1,770 @@
+import { style } from '../../packages/@react-spectrum/s2/style/spectrum-theme' with {type: 'macro'};
+import {Link} from '@react-spectrum/s2';
+import {P, Code, Pre, H3, H2} from './typography';
+
+export function Migrating() {
+  return (
+    <div className={style({maxWidth: 'lg', marginX: 'auto', fontFamily: 'sans'})}>
+      <div className={style({marginX: 48})}>
+        <h1 className={style({fontSize: 'heading-2xl', color: 'heading', marginBottom: 48})}>
+          Migrating to Spectrum 2
+        </h1>
+        <P>An automated upgrade assistant is available by running the following command in the project you want to upgrade:</P>
+        <Pre>npx @react-spectrum/codemods s1-to-s2</Pre>
+
+        <P>For cases that the upgrade assistant doesn't handle automatically or where you'd rather upgrade some components manually, use the guide below.</P>
+
+        <P>Note that [PENDING] indicates that future changes will occur before the final release, and the current solution should be considered temporary.</P>
+
+        <H2>Components</H2>
```

Comment:
Is updating this section going to be hard/manual work or was this 1:1 from the diff script? If it was manually created maybe we could just point to somewhere we host the output of the diff script instead? Also just worried about how to inform users of additional updates or changes to the current stated props (e.g. if a decision is reversed/name is changed etc)

Reply:
IMO this is easier to consume than the API diff. Some actions don't translate well to the diff, and this eliminates some noise. To maintain, we just need to look at the API diff when release and update this with any breaking changes.

Reply:
sounds good